requirements-test.txt

```diff
@@ -1,3 +1,4 @@
+coveralls
```

Comment:
not needed, openwisp-utils brings this dependency in: https://github.com/openwisp/openwisp-utils/blob/master/requirements-test.txt#L1

Reply:
But that's in `requirements-test.txt` :sweat_smile: